app/controllers/configuration_controller.rb

```diff
@@ -530,11 +530,7 @@ def set_form_vars
       }
 
       current_tz = @edit.fetch_path(:current, :display, :timezone)
-      if current_tz.blank?
-        new_tz = MiqServer.my_server.get_config("vmdb").config[:server][:timezone]
-        new_tz = 'UTC' if new_tz.blank?
-        @edit.store_path(:current, :display, :timezone, new_tz)
-      end
+      @edit.store_path(:current, :display, :timezone, ::Settings.server.timezone) if current_tz.blank?
```

Comment:
feels like we should be using `Time.zone` (probably out of scope for this PR though)